client/app/views/steps/password.coffee

```diff
@@ -1,24 +1,63 @@
 StepView = require '../step'
 
+
 module.exports = class PasswordView extends StepView
 
     template: require '../templates/view_steps_password'
 
     events:
-        'click button': 'onSubmit'
+        'click .next': 'onSubmit'
+        'click [action=password-visibility]': 'onToggleVisibility'
+
+
+    isVisible: false
+
+
+    renderInput: =>
+        data = @serializeInputData()
+
+        # Show/hide password value
+        @$('input[name=password]').attr 'type', data.inputType
```

Comment:
Calling $ is pretty expensive. As a good practice we should cache those results in view's inner variables. See: https://ttmm.io/tech/selector-caching-jquery/
This can be done at the end of `render()` method or the first time `renderInput` is called for example.